activation/activation.go

```diff
@@ -39,14 +39,22 @@ var (
 
 // PoetConfig is the configuration to interact with the poet server.
 type PoetConfig struct {
-	PhaseShift                     time.Duration `mapstructure:"phase-shift"`
-	CycleGap                       time.Duration `mapstructure:"cycle-gap"`
-	GracePeriod                    time.Duration `mapstructure:"grace-period"`
-	RequestTimeout                 time.Duration `mapstructure:"poet-request-timeout"`
-	RequestRetryDelay              time.Duration `mapstructure:"retry-delay"`
+	DefaultRequestTimeout time.Duration `mapstructure:"poet-client-default-request-timeout"`
+	RequestRetryDelay     time.Duration `mapstructure:"poet-client-retry-delay"`
+	CertifierInfoCacheTTL time.Duration `mapstructure:"poet-client-certifier-info-cache-ttl"`
+	MaxRequestRetries     int           `mapstructure:"poet-client-retry-max"`
+	// Start of new PoET round
+	PhaseShift time.Duration `mapstructure:"phase-shift"`
+	// A gap between end of old PoET round and start of new one
+	CycleGap time.Duration `mapstructure:"cycle-gap"`
+	// Time in the end of cycle gap, when PoST challenge must be build and send to PoET server
+	GracePeriod time.Duration `mapstructure:"grace-period"`
+	// Period to find positioning ATX. Must be less, than GracePeriod
 	PositioningATXSelectionTimeout time.Duration `mapstructure:"positioning-atx-selection-timeout"`
-	CertifierInfoCacheTTL          time.Duration `mapstructure:"certifier-info-cache-ttl"`
-	MaxRequestRetries              int           `mapstructure:"retry-max"`
+	// Period to submit PoST challenge to PoET server. Must be not greater than GracePeriod
+	SubmitChallengeTimeout time.Duration `mapstructure:"submit-challenge-timeout"`
+	// Period to get PoET proof
+	GetProofTimeout time.Duration `mapstructure:"get-proof-timeout"`
```

Comment:
```suggestion
	FetchProofTimeout time.Duration `mapstructure:"fetch-proof-timeout"`
```

Regarding naming for mapstructure:  the submit and fetch timeout should probably also get the `poet-client-` prefix.

Reply:
Idea was use poet-client prefix for parameters, which are universal for all poet client requests